recipe/meta.yaml

```diff
@@ -21,7 +21,7 @@ build:
   number: 3
```

Comment:
Now that we're unconditionally removing the `cudatoolkit` run-dependence (even though it should generally get replaced by a run-export where relevant), it's probably safer to increment the build number.